Template/ClientCPP.tt

```diff
@@ -17,17 +17,30 @@ foreach(GrpcService service in s.ServiceArray)
 foreach (GrpcServiceMethod method in service.MethodArray)
 {
 #>
-FGrpcContextHandle U<#=service.Name#>Client::Init<#=method.Name#>()
```

Comment:
nit: Maybe we should keep the old Init() function for backwards-compatiblity? That way we should be able to upstream this as well